### PR TITLE
ENH code to munge missing R licenses

### PR DESCRIPTION
**Pull request by beckermr**
This PR adds special munging for R licenses.

### CODE REVIEW

**Comment by beckermr** [comment]
@isuruf @CJ-Wright LMK if you have comments. I won't be able to merge till later, so go ahead if you think it is good and the tests are passing.

**Comment by beckermr** [comment]
This test is failing: "tests/test_license_migrator.py::test_version_license_correct FAILED      [ 34%]
"

IDK why and I have to run. Feel free to push a fix.

File: conda_forge_tick/utils.py [hunk 1]
```diff
@@ -68,6 +68,17 @@
 )
 
 
+# https://stackoverflow.com/questions/6194499/pushd-through-os-system
+@contextlib.contextmanager
+def pushd(new_dir):
```
Comment (CJ-Wright) [inline conda_forge_tick/utils.py:73]: why not use `indir`?

Comment (beckermr) [reply conda_forge_tick/utils.py:73]: Because I didn’t know it existed.

Comment (CJ-Wright) [reply conda_forge_tick/utils.py:73]: It's used elsewhere in the code

Comment (beckermr) [reply conda_forge_tick/utils.py:73]: Cool!

**Comment by CJ-Wright** [comment]
@beckermr is this ready to go?

**Comment by beckermr** [comment]
Do we want to run the license stuff for all R packages or just those that are missing a license file?

Right now we just run on those missing a file. We could run everything to make sure things are updated.

File: conda_forge_tick/utils.py (hunk 1 above)
Comment (beckermr) [reply conda_forge_tick/utils.py]: fixed

**Pull request merged**